core/src/main/java/com/amplifyframework/predictions/models/Label.java

```diff
@@ -120,5 +146,10 @@ public Label build() {
         List<String> getParentLabels() {
             return Objects.requireNonNull(parentLabels);
         }
+
+        @NonNull
+        List<RectF> getBoxes() {
+            return Objects.requireNonNull(boxes);
```

Comment:
```suggestion
            return Objects.requireNonNull(Immutable.of(boxes));
```

Reply:
good call! will apply this to the other variable in this class too.